### DISCUSSION TITLE
Issue when using Quad9

### DISCUSSION THREAD

**Discussion by naufalprojects** [AdguardTeam/AdGuardHome#2792] [Q&A] [answered]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x ] I am running the latest version
- [x ] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.105.1 
* **How did you install AdGuard Home:**
  * From Github "sh" autoinstall
* **How did you setup DNS configuration:**
  * VPS
* **CPU architecture:**
  * X86_64
* **Operating system and version:**
  * Ubuntu 18.04.1

### Expected Behavior
I am running AdGuard Home, and for the upstream is Quad9. The problem is, when i do a "test upstream" it shown "could not be used..." (Screenshot below this) Also when i do a browsing, the browser shown "ERR_NAME_NOT_RESOLVED" But, when i tried on secondary server (same region) and it worked, no problem

And, tried to using CloudFlare as an upstream, no problem. It worked normally (on primary server, routed to Indonesia)

Btw, Quad9 DNS in my primary VPS routed to Indonesia. And my secondary server, routed to Singapore. Both the server located in Indonesia

Screenshot : 
![Screenshot_20210309-062435](https://user-images.githubusercontent.com/52203960/110453376-aeedd900-80f8-11eb-996a-ce380a118b1c.png)

**Comment by ameshkov** [comment, marked as answer]
Moved to Discussions since this is not a bug.

It looks like `dns10.quad9.net` is simply not reachable from your place.

If you want to troubleshoot it, you may try using https://github.com/ameshkov/dnslookup, do some lookups to quad9 and see what error it shows:
`dnslookup example.org tls://dns10.quad9.net`

**Reply by naufalprojects** [reply]
Here is the results :
Edit : i tested on primary server

![image](https://user-images.githubusercontent.com/52203960/110454382-bcf02980-80f9-11eb-9e5d-4b98b6abc56f.png)

**Reply by ameshkov** [reply]
Well, as I've said, Quad9 is not reachable from your server. No idea why, but I guess you shouldn't use it as an upstream.

**Reply by naufalprojects** [reply]
Ah, okay 😢 
Thx for ur explain.
